pkgs/applications/misc/far2l/default.nix

```
@@ -1,29 +1,23 @@
 { lib, stdenv, fetchFromGitHub, fetchpatch, makeWrapper, cmake, pkg-config, wxGTK30, glib, pcre, m4, bash,
-  xdg-utils, gvfs, zip, unzip, gzip, bzip2, gnutar, p7zip, xz, imagemagick, darwin }:
+  xdg-utils, gvfs, zip, unzip, gzip, bzip2, gnutar, p7zip, xz, imagemagick, darwin,
```

```suggestion
  xdg-utils, gvfs, zip, unzip, gzip, bzip2, gnutar, p7zip, xz, imagemagick, Cocoa,
```

and inherit it from top level.

---

Looks like it's optional? What would be the correct way to request an optional package here?
`Cocoa ? []` ?

---

Works for me. Ready for merge if so.

---

> Looks like it's optional? What would be the correct way to request an optional package here?
Cocoa ? [] ?

```
buildInputs = lib.optionals stdenv.isDarwin [ Cocoa ];
```

---

Nae, I mean in function arguments. _Anonymous function called without required argument 'Cocoa'_. See current code in the patch.